src/components/ContentNode.vue

```diff
@@ -267,8 +263,12 @@ function renderNode(createElement, references) {
         return renderFigure(node);
       }
 
-      return createElement(Table, {}, (
-        renderTableChildren(node.rows, node.header)
+      return createElement(Table, {
+        props: {
+          spanned: !!node.extendedData,
```

Comment:
Is this still needed since the class isn't being used anymore?

Reply:
Probably not, just thought it might help future devs, if they want to style these tables in some way with custom styling 🤔